tcf_core/settings/base.py

```diff
@@ -7,7 +7,10 @@
 import environ
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
idk if this is really necessary but this entire file had formatting things that were changed but it might be better to delete these changes to keep it consistent with how it was before.